### PR TITLE
Ga/sigil post

### PR DESCRIPTION
**Pull request by g-a-v-i-n**
- copy edits
- new assets
- make assets square
- make arena embeds square

### CODE REVIEW

File: content/blog/creating-sigils.md [hunk 1]
```diff
@@ -68,7 +69,7 @@ You can see the whole collection of our visual research [here](https://www.are.n
 
 Ultimately, we didn’t specifically draw on any of these influences very consciously. The goal of visual research is to get a sense of the territory, what has been tried, and what works. We think these threads of prior work are evident in the final result — you can *feel* them, but it’s tough to point to them individually.
 
-## What’s in a name?
+## What’s in a Name?
```
Comment (matildepark) [inline content/blog/creating-sigils.md]: I am pretty sure we use sentence casing. @collinscorrina ?

**Review by matildepark — APPROVED** [review]
Everything else looks fine pending her take on the case.

File: content/blog/creating-sigils.md (hunk 1 above)
Comment (crrncllns) [reply content/blog/creating-sigils.md]: Yes, you're right. @g-a-v-i-n can you update?

Comment (g-a-v-i-n) [reply content/blog/creating-sigils.md]: you got it

**Review by crrncllns — APPROVED** [review]
lgtm!

**Pull request merged**